### ISSUE TITLE
Netflix won't play, it needs WidevineCdm

### ISSUE THREAD

**Issue by freefrancisco** [jiahaog/nativefier#70]
I built an app for netflix, but when I try to play a movie it gives me this error: 
````
Whoops, something went wrong...

Missing Component

We cannot find all the required components to play Netflix on this device. 
Please visit chrome://components, locate the WidevineCdm component, and click the "Check for update" button.
For additional assistance resolving this issue, please visit our Help Center.
https://help.netflix.com/support/27451
````
Any ideas on how to add this plugin?

**Comment by jiahaog** [comment]
This is a duplicate of #68

**Comment by jiahaog** [comment]
I'll look into it soon!

**Comment by quanglam2807** [comment]
@freefrancisco  It has been fixed in WebCatalog webcatalog/desktop#4

**Comment by RickSlashee** [comment]
Hi there, I am having the same issue with Linux Mint, any idea?

![screenshot from 2019-02-01 13-38-51](https://user-images.githubusercontent.com/40864652/52145546-e0953b80-2626-11e9-9d35-350bcff0b1bb.png)
